acceptance/setup.sh

```diff
@@ -40,6 +40,7 @@ echo "ZITADEL_API_URL=${ZITADEL_API_URL}
 ZITADEL_SERVICE_USER_ID=${ZITADEL_SERVICE_USER_ID}
 ZITADEL_SERVICE_USER_TOKEN=${PAT}
 SINK_NOTIFICATION_URL=${SINK_NOTIFICATION_URL}
+EMAIL_VERIFICATION=true
```

Comment:
where is this used?

Reply:
https://github.com/zitadel/typescript/blob/main/apps/login/src/lib/verify-helper.ts#L68